spark/dl/src/test/scala/com/intel/analytics/bigdl/keras/KerasRunner.scala

```diff
@@ -109,11 +109,18 @@ object KerasRunner {
                               keyName: String): Array[Tensor[Float]] = {
     if (!pvalues.keySet.filter(key => key.contains(keyName)).isEmpty) {
       val weightNum = pvalues.keySet.filter(key => key.contains(keyName)).size / 2
-      Range(0, weightNum).map {i =>
-        Tensor[Float](
-          data = pvalues(s"${keyName}_${i}_value"),
-          shape = pvalues(s"${keyName}_${i}_shape").map(_.toInt))
-      }.toArray
+      if (weightNum > 1) { // with bias
```

Comment:
any specific reason for this changes?

Reply:
Yes. If the layer has no bias,
`data = pvalues(s"${keyName}_${i}_value")` will get an error as there's only 1 weight, it will be just `${keyName}` instead of `${keyName}_0`